### ISSUE TITLE
Citation problems in Sphinx docs

### ISSUE THREAD

**Issue by ksbeattie** [CCSI-Toolset/FOQUS#389]
When generating the docs I see the following warnings:
```
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_opt/reference/optimization.rst:4: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_opt/reference/optimization.rst:4: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_opt/reference/optimization.rst:4: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_opt/tutorial/parameter_est.rst:224: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_solventfit/reference.rst:3: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_surrogates/tutorial/acosso.rst:6: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_surrogates/tutorial/acosso.rst:6: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_surrogates/tutorial/bssanova.rst:6: WARNING: Unknown interpreted text role "raw-latex".
/home/ksb/Projects/CCSI/github/eslickj/foqus/docs/source/chapt_surrogates/tutorial/bssanova.rst:6: WARNING: Unknown interpreted text role "raw-latex".
```
When looking at that first page, I see:
![image](https://user-images.githubusercontent.com/1534843/52753166-7ecfbc80-2faa-11e9-8179-ea8b78f8c3e3.png)

So those citations just need to be fixed.

**Comment by ksbeattie** [comment]
Given that there are so few of these, we might just write them out as links to a references section of the document.

**Comment by eslickj** [comment]
Fixed in PR #410, just added a references section and linked the citations to that.
